app/scripts/modules/ecs/src/serverGroup/configure/wizard/capacityProvider/CapacityProvider.tsx

```diff
@@ -61,6 +101,19 @@ class EcsCapacityProvider extends React.Component<IEcsCapacityProviderProps, IEc
     this.setState({ capacityProviderStrategy: capacityProviderStrategy });
   };
 
+  private updateCapacityProviderStrategy = (targetCapacityProviderType: boolean) => {
```

Comment:
Can you rename this to updateCapacityProviderType (this isn't updating the strategy but rather they type of the strategy)?

Reply:
Fixed it.